web/src/api/generated/services/HealthCheckService.ts

```diff
@@ -20,7 +20,7 @@ export class HealthCheckService {
                 401: `Unauthorized`,
                 403: `Forbidden`,
                 404: `Not Found`,
-                422: `Unprocessable Entity`,
+                422: `Unprocessable Content`,
```

Comment:
Where are these generated from? How can the value change when nothing has changed in the python code?

Reply:
I would guess it's due to a version change in the generator.